src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -623,8 +628,8 @@ class ScenarioRepository final : public IScenarioRepository
             for (uint32_t i = 0; i < numHighscores; i++)
             {
                 ScenarioHighscoreEntry* highscore = InsertHighscore();
-                highscore->fileName = fs.ReadString();
-                highscore->name = fs.ReadString();
+                highscore->fileName = fs.ReadStdString();
```

Comment:
Will this cause invalidation of the file format? Or does it just return a std string but internally its the same.

Reply:
Both versions wrote a null terminated sequence, this just results in std::string